app/components/ChatView.js

```diff
@@ -0,0 +1,157 @@
+import React, { Component } from 'react';
+ import { Text,View,StyleSheet,Image} from 'react-native';
+import PropTypes from 'prop-types'
+import SvgUri from 'react-native-svg-uri'
+
+import { sendMessage, startChat } from '../redux/chat';
+
+ class ChatView extends Component{
+  constructor(props){
+    super(props)
+    this.state ={
```

Comment:
Do we need this state?

Reply:
when we need to input some message. We may need to change the state